paper_framework_sources/p1897_initial_executor_algorithms_for_cpp23.md

```diff
@@ -501,24 +544,35 @@ float r =
       [](int a, float b){return a + b;}));
 // r==4.2
 ```
+*- end example]*
 
 ### Wording
-
 The name `execution::when_all` denotes a customization point object.
-The expression `execution::when_all(S)` for some subexpression `S` is expression-equivalent to:
+For some subexpression `ss`, let `Ss` be a list of types such that `decltype((ss))...` is `Ss...`.
+The expression `execution::when_all(ss...)` is expression-equivalent to:
 
- * Otherwise, `when_all(S)` if that expression is valid with overload resolution performed in a context that includes the declaration
+ * `when_all(ss...)` if that expression is valid, and if `S` satisfies `sender` and if `Sch` satisfies `scheduler`, with overload resolution performed in a context that includes the declaration
  ```
-         template<class S>
-           void when_all(S) = delete;
+      void when_all() = delete;
  ```
+   and that does not include a declaration of `execution::when_all`.
+
+ * Otherwise constructs a receiver, `ri` for each passed `sender` `si` in `ss` and passes that receiver to `execution::connect(si, ri)`, resulting in an `operation_state` `osi` such that:
+
+   When some `output_receiver` has been passed to `connect` on the returned `sender`.
```

Comment:
I would expect to see that each ri forwards get_scheduler through to output_receiver.

Reply:
It does in P1898R1.